Disc/Networking/APIClient.swift

```diff
@@ -12,4 +12,10 @@ public struct APIClient {
     public init(token: String) {
         self.token = token
     }
+    
+    /// Create an APIClient with no access token.
+    ///
+    public init() {
+        self.token = ""
+    }
```

Comment:
For the log in we don't have a bearer token at that stage, as opposed to the add and remove identity. Should we make it optional, for init with an empty string? Not sure how to address that.